niworkflows/anat/ants.py

```diff
@@ -252,13 +252,20 @@ def init_brain_extraction_wf(name='brain_extraction_wf',
         init_aff.inputs.search_grid = (40, (0, 40, 40))
 
     # Set up spatial normalization
-    norm = pe.Node(Registration(
-        from_file=pkgr_fn(
-            'niworkflows.data',
-            'antsBrainExtraction_%s.json' % normalization_quality)),
-        name='norm',
-        n_procs=omp_nthreads,
-        mem_gb=mem_gb)
+    if use_laplacian:
+        norm = pe.Node(Registration(from_file = pkgr_fn(
```

Comment:
just as a stylistic suggestion, the if ... else can be more readable stripping the settings file name:

```Python
    settings_file = 'antsBrainExtraction_%s.json' if use_laplacian \
        else 'antsBrainExtractionNoLaplacian_%s.json'
    norm = pe.Node(Registration(from_file = pkgr_fn(
            'niworkflows.data', settings_file % normalization_quality)),
        mem_gb=mem_gb,
        n_procs=omp_nthreads,
        name='norm')
```